packages/react-native-quick-crypto/QuickCrypto.podspec

```diff
@@ -17,7 +17,32 @@ Pod::Spec.new do |s|
   s.macos.deployment_target = 10.13
   s.tvos.deployment_target = 13.4
 
-  s.source       = { :git => "https://github.com/margelo/react-native-quick-crypto.git", :tag => "#{s.version}" }
+  s.source = { :git => "https://github.com/margelo/react-native-quick-crypto.git", :tag => "#{s.version}" }
+
+  # cocoapod for Sodium has not been updated for a while, so we need to build it ourselves
+  # https://github.com/jedisct1/swift-sodium/issues/264#issuecomment-2864963850
+  s.prepare_command = <<-CMD
+    # Create ios directory if it doesn't exist
+    mkdir -p ios
+
+    # Download libsodium
+    curl -L -s -o ios/libsodium.tar.gz https://download.libsodium.org/libsodium/releases/libsodium-1.0.20-stable.tar.gz
```

Comment:
Is this the download you don't like? as mentioned in PR description :vomiting_face: